Source/scrollrt.cpp

```diff
@@ -148,20 +155,30 @@ static void scrollrt_draw_cursor_item()
 	assert(!sgdwCursWdt);
 
 	if (pcurs <= CURSOR_NONE || cursW == 0 || cursH == 0) {
+		if (hwcursor)
+			SDL_ShowCursor(SDL_DISABLE);
 		return;
 	}
 
 	if (sgbControllerActive && !IsMovingMouseCursorWithController() && pcurs != CURSOR_TELEPORT && !invflag && (!chrflag || plr[myplr]._pStatPts <= 0)) {
+		if (hwcursor)
+			SDL_ShowCursor(SDL_DISABLE);

```

Comment:
Since this function is called basically every frame, I didn't feel like spamming the log with the same error message was such a great idea.

Reply:
It would be good if we could limit this to only be called on state change.

Reply:
Yes, ideally. But if actually peek at SDL_SetCursor's code you can see that it actually does state tracking of its own so it's really a very cheap call. Sure we can add another state tracker on top of that but it would be prone to desync of the two states.